pkg/sdkserver/sdkserver.go

```diff
@@ -192,8 +193,21 @@ func (s *SDKServer) Run(stop <-chan struct{}) error {
 	}
 
 	// grab configuration details
+	if gs.Spec.Logging.SdkServer == "" {
+		s.logLevel = "info"
+	} else {
+		s.logLevel = strings.ToLower(gs.Spec.Logging.SdkServer)
+	}
+	s.logger.WithField("logLevel", s.logLevel).Info("Setting LogLevel configuration")
+	level, err := logrus.ParseLevel(s.logLevel)
+	if err == nil {
+		s.logger.Logger.SetLevel(level)
+	} else {
+		s.logger.WithError(err).Info("Specified wrong Logging.SdkServer")
```

Comment:
what log level is actually being used by logrus in this case? I assume it's info, in which case we should also set s.logLevel to "info" so that they are consistent.

Reply:
on second thought, is there a reason that we store loglevel in the struct? why not make it a local variable?

Reply:
yes, you are right it should be simplified

Reply:
doing it as a member of struct was intended to support changing the logLevel on the go, by configuring GS Spec.